posthog/permissions.py

```diff
@@ -216,3 +216,14 @@ def has_permission(self, request, view) -> bool:
             else OrganizationMembership.Level.ADMIN
         )
         return requesting_level >= minimum_level
+
+
+class StaffUser(BasePermission):
+    """
+    Allows access to only staff users
+    """
+
+    message = "You are not a staff user, contact your instance admin."
+
+    def has_permission(self, request, _):
+        return request.user.is_staff
```

Comment:
Awesome - but users are not default staff on new installs. We should probably make the first user (installer) staff by default

Reply:
Great point, will do that

Reply:
done